uml4net.xmireader/Readers/StructuredClassifiers/ClassReader.cs

```diff
@@ -103,7 +103,8 @@ public override IClass Read(XmlReader xmlReader)
                 this.Cache.Add(@class.XmiId, @class);
 
                 @class.Name = xmlReader.GetAttribute("name");
-
+                @class.IsAbstract = bool.TryParse(xmlReader.GetAttribute("isAbstract"), out var isAbstract) && isAbstract;
```

Comment:
to keep the patterns simple, and later the code gen template simple, can i suggest the use the following syntax?


```
var isAbstract = xmlReader.GetAttribute("isAbstract");
if (!string.IsNullOrEmpty(isAbstract))
{
    @class.IsAbstract = bool.Parse(isAbstract);
}

```

very similar to how the `visibility` property is processed